templates/statefulset-zabbix-proxy.yaml

```diff
@@ -87,7 +91,12 @@ spec:
 {{- toYaml .Values.zabbixproxy.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if .Values.zabbixproxy_image.tag }}
+          image: "{{ .Values.zabbixproxy.image.repository }}:{{ .Values.zabbixproxy_image.tag }}"
```

Comment:
Hi @scoopex!

Can you change

`image: "{{ .Values.zabbixproxy.image.repository }}:{{ .Values.zabbixproxy_image.tag }}"`

by

`image: "{{ .Values.zabbixproxy.image.repository }}:{{ .Values.zabbixproxy.image.tag }}"`

?

Reply:
Sure, done :-)